server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableUtils.java

```diff
@@ -534,10 +540,10 @@ public static AccumuloConfiguration getCompactionConfig(CompactionKind kind, Tab
   }
 
   static StoredTabletFile compact(Tablet tablet, CompactionJob job, Set<StoredTabletFile> jobFiles,
-      Long compactionId, boolean propogateDeletes, CompactableImpl.CompactionHelper helper,
-      List<IteratorSetting> iters, CompactionCheck compactionCheck, RateLimiter readLimiter,
-      RateLimiter writeLimiter, CompactionStats stats)
-      throws IOException, CompactionCanceledException {
+      Long compactionId, Set<StoredTabletFile> selectedFiles, boolean propogateDeletes,
```

Comment:
```suggestion
      Long compactionId, Set<StoredTabletFile> selectedFiles, boolean propagateDeletes,
```

Reply:
Looks like there are lots of occurrences of this typo actually. propogate vs propagate

Reply:
@DomGarguilo I ran

```bash
find . -name "*.java" | xargs sed -i '' -e 's/propogate/propagate/g'
```

 and pushed the changes in d90f42178f76171e785372e0bf1a04359d24e816